packages/babel-preset-env/src/filter-items.js

```diff
@@ -0,0 +1,78 @@
+import semver from "semver";
```

Comment:
(NOTE for other reviewers)

This file isn't new, it has just been extracted from `babel-preset-env/src/index.js`